api-report/runtime-definitions.api.md

```diff
@@ -378,6 +384,16 @@ export type NamedFluidDataStoreRegistryEntry = [string, Promise<FluidDataStoreRe
 // @public (undocumented)
 export type SummarizeInternalFn = (fullTree: boolean, trackState: boolean) => Promise<ISummarizeInternalResult>;
 
+// @public
+enum VisibilityState_2 {
+    GloballyVisible = "GloballyVisible",
```

Comment:
How about Connected state - `GloballyConnected`, `LocallyConnected`, `Disconnected`

Reply:
We already have connected state which means something totally different. This is more about the data store being visible or reachable within a container and / or from all clients in the document.